### ISSUE TITLE
Provide an alternative to prerecorded audio/video

### ISSUE THREAD

**Issue by adamjarling** [avalonmediasystem/structured-metadata-editor-react#67]
### Description
SME audit report line item 1.2.1, referenced in this issue: #64

WCAG 2.1 Audio-only and Video-only (Prerecorded).  Provide an alternative to prerecorded audio/video.

### Done Looks Like
- This audit test passes in whichever tool described the failure.

To Do
- [ ]  Determine what is a valid acceptance criteria for this situation? What is a reasonable alternative here?
- [ ] Investigate what other applications do here to solve this issue.
- [ ] Implement a solution
- [ ] Re-test.
